### ISSUE TITLE
Broken Link on Accessibility Page

### ISSUE THREAD

**Issue by AJSMITH1512** [nhsuk/nhsuk-service-manual#958]
### What

There is a broken link on the Accessibility pages that should be fixed.

Clicking on the link gives 404

Need to identify relevant page to link to

### Specific issue point

**Content**

**Use headings correctly**

In the drop down - Structure headings for accessibility

You can use a web developer toolbar (Chrome or Firefox) to see the overall heading structure of the page.

Chrome link broken - this is also on Design and Testing pages.

### To do

- [ ] identify correct link
- [ ] Correct link for each page - Content, Design and Testing Pages
- [ ] Release update

**Comment by Tosin-Balogun** [comment]
@davidhunter08 - I don't think we got to discuss this last week boss
@AdamChrimes - Do you know the details about this issue?

**Comment by AdamChrimes** [comment]
@Tosin-Balogun I don't, might be worth speaking to @AJSMITH1512 to find out where the broken link is so that it can be fixed

**Comment by Tosin-Balogun** [comment]
# Update

I have found the bug, please see image below. I think is a good one for me to try fix 😄 @davidhunter08 @AdamChrimes 


<img width="883" alt="Screenshot 2021-03-01 at 10 58 35" src="https://user-images.githubusercontent.com/71393402/109488722-d5b76a00-7a7d-11eb-8f1e-f0a58286a79b.png">

**Comment by sarawilcox** [comment]
LGTM. Well done @Tosin-Balogun.
